commands/http/handler.go

```diff
@@ -261,7 +261,9 @@ func sendResponse(w http.ResponseWriter, r *http.Request, res cmds.Response, req
 	h.Set("Access-Control-Allow-Headers", "X-Stream-Output, X-Chunked-Output")
 	// expose those headers
 	h.Set("Access-Control-Expose-Headers", "X-Stream-Output, X-Chunked-Output")
-
+	// Expose our agent to allow identification
+	h.Set("Server", "go-ipfs/" + config.CurrentVersionNumber)
```

Comment:
Should call Set earlier, ideally before the first possible response, which is right at the start of the function